swap/src/asb/config.rs

```diff
@@ -250,6 +254,11 @@ pub fn query_user_for_initial_config(testnet: bool) -> Result<Config> {
         .default(defaults.monero_wallet_rpc_url)
         .interact_text()?;
 
+    let kraken_url = Input::with_theme(&ColorfulTheme::default())
+        .with_prompt("Enter Kraken Price Ticker WebSocket URL or hit enter to use default")
+        .default(defaults.kraken_ws_url)
+        .interact_text()?;
+
```

Comment:
Personally I would prefer this to not to be in the wizard. 
If it is here, an ASB provider gets confronted with this setup not knowing what to do. 
I think it's more of an expert setup and configuring it via the config file directly is enough.

Reply:
Agree with this, it is unlikely this needs to be configured on the first start.

Reply:
https://github.com/comit-network/xmr-btc-swap/pull/585/commits/305f02a672b071582786f29ab1e8dac576f4fe4a